csrc/device_lower/pass/inline_ptx.cpp

```diff
@@ -19,6 +19,14 @@
 namespace nvfuser {
 
 class LowerToInlinePtx : public kir::ExprMutator {
+ private:
```

Comment:
Could you split this change out as a separate PR? Moving `invertedPredicate` looks fine to me, but it generates lot of diff in the generated code, I want to run codegen diff in this PR to get the impact, but this change is creating lots of noise in that output.